### DISCUSSION TITLE
How does a gpu cluster system like SLRUM use ddp training ?

### DISCUSSION THREAD

**Discussion by takfate** [Lightning-AI/pytorch-lightning#6715] [DDP / multi-GPU / multi-node] [answered]
I need to use `srun` run `python, so how does set Trainer of `pl` correctly?

**Comment by awaelchli** [comment, marked as answer]
Not sure what you are asking. Is the question how to use PL with SLURM?
I can point you to the SLURM tutorial in our docs:
https://pytorch-lightning.readthedocs.io/en/latest/clouds/slurm.html
